Tools/TestWebKitAPI/Tests/mac/ContextMenuTests.mm

```diff
@@ -468,7 +468,12 @@ static void swizzledOrderFrontColorPanel(id, SEL, id)
     EXPECT_WK_STREQ(elementInfo.qrCodePayloadString, "https://www.webkit.org");
 }
 
+// FIXME: Re-enable this test once webkit.org/b/266650 is resolved
+#if PLATFORM(MAC) && (__MAC_OS_X_VERSION_MIN_REQUIRED > 140000) && !defined(NDEBUG)
```

Comment:
style nit: you don't need the parentheses around `__MAC_OS_X_VERSION_MIN_REQUIRED`